spec/models/profile_field_spec.rb

```diff
@@ -20,5 +20,17 @@
       field = create(:profile_field, label: "Test? Test! 1")
       expect(field.attribute_name).to eq "test_test1"
     end
+
+    describe "#maximum_header_field_count" do
+      it "limits the number of header fields" do
+        count = [0, described_class::HEADER_FIELD_LIMIT - described_class.header.count].max
```

Comment:
While this isn't great to read it made the spec less error-prone as now it doesn't matter how many header fields already exist when running this.